css/main.css

```diff
@@ -383,6 +383,16 @@ content {
 
 .logo {
   width: 200px;
+  position: absolute;
+  left: 1rem;
```

Comment:
Don’t use rem here, use % instead

Reply:
I did but I don't understand why and when to use what...doesnt rem also scale dynamically?